gallery/visualtests/testUtils.js

```diff
@@ -56,15 +56,35 @@ module.exports = {
 
     beforeEach(async function() {
       page = await browser.newPage();
+
       await page.goto(pageUrl + pageFragmentIdentifier);
 
+      // eslint-disable-next-line no-undef
```

Comment:
This shouldn't be needed now
```suggestion
```

Reply:
> This shouldn't be needed now

thanks for catching that, updated here [fac266c](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/fac266cd71d7dacb045c29ca4b52629201bf25db)